lib/rouge/lexers/powershell.rb

```diff
@@ -214,14 +214,16 @@ class Powershell < RegexLexer
 
         rule %r/-{1,2}\w+/, Name::Tag
 
+        rule %r/[12]>/, Operator
+
         rule %r/(\.)?([-\w]+)(\[)/ do |m|
           groups Operator, Name, Punctuation
           push :bracket
         end
 
         rule %r/([\/\\~[a-z]][-.:\/\\~\w]*)(\n)?/i do |m|
           groups Name, Text::Whitespace
-          push :parameters
+          push :parameters unless m[3].nil?
```

Comment:
There are only 2 captured groups so we can only access `m[0]` and `m[1]`. Hence this condition is a no-op I think 🤔